### DISCUSSION TITLE
Debugging the async onSubmit() function

### DISCUSSION THREAD

**Discussion by mjstelly** [jaredpalmer/formik#3667] [Q&A]
On my form, I have a bug in the `onSubmit()` function, but this function is asynchronous. How can I log what's happening inside the `onSubmit()` execution?
```
onSubmit={async (values) => {
    console.log('formik onSubmit values: ', values)
}
```
For instance, I trigger the submit with `props.handleSubmit();`. Once execution reaches the function, log output is not displayed on the console. Please ask if you have questions, or if my explanation is unclear.

**Comment by Thoobi** [comment]
hello mate, did you figure it out?
